### ISSUE TITLE
FloatingActionButton not displaying the shadow

### ISSUE THREAD

**Issue by raylee4204** [fafaldo/FABToolbar#3]
The native FloatingActionButton has elevation property, which doesn't seem to be displayed using this library

**Comment by fafaldo-zz** [comment]
This issue is caused by the fact that I use `setLayerType(SOFTWARE)` on a fab container. It basically cancels the native shadow (Lollipop and up) but does not impact support versions (lower APIs).
The reason for this is because I encountered a strange bug on API level 21, when expanding FAB makes strange artifacts in the FAB container. You should probably be able to reproduce it by setting layer type manually back to other value, like:

```
fabContainer.setLayerType(View.LAYER_TYPE_NONE, null);
or
fabContainer.setLayerType(View.LAYER_TYPE_HARDWARE, null);
```

Again, this didn't happen to me with any other API versions.
I played with that a bit, and found that setting layerType to SOFTWARE just for FAB (instaed of the whole container) is enough, and it both - solves the artifacts issue and restores the shadow. I am currennly not able to test that on more devices though, so it will be changed after I confirm it to be working on all the devices I can get my hands on (probably very soon though). For now, you can try changing container layerType back to NONE/HARDWARE and setting FAB layerType to SOFTWARE.

**Comment by fafaldo-zz** [comment]
fixed in 1.1.0
